mslib/mscolab/server.py

```diff
@@ -375,7 +375,7 @@ def upload_profile_image():
 @APP.route('/fetch_profile_image', methods=["GET"])
 @verify_user
 def fetch_profile_image():
-    user_id = request.form['user_id']
+    user_id = g.user.id
```

Comment:
This doesn't seem right. On upload it makes sense, to ensure that a user can only upload a profile image for themselves. But on fetch this would lead to every user being shown with the same profile image - the one belonging to the user who fetched the images.

Reply:
Ahh okay